great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -18,10 +18,9 @@ def __init__(
         name: str,
         domain_builder: Optional[DomainBuilder] = None,
         parameter_builders: Optional[List[ParameterBuilder]] = None,
-        expectation_configuration_builders: Optional[
-            List[ExpectationConfigurationBuilder]
+        expectation_configuration_builders: List[
+            ExpectationConfigurationBuilder
         ] = None,
-        variables: Optional[ParameterContainer] = None,
```

Comment:
I don't believe we should modify our source code in order to get our tests to work. I'd opt to modify the tests in question to work with the actual behavior.

I think type annotating this without the `Optional` signifies to the reader that this will always be a defined list when used (which is the true behavior when schema validated). I think that's an important bit of information we don't want to lose.

Reply:
@cdkini Fixed -- thanks!